cogs5e/models/spell.py

```diff
@@ -447,6 +449,13 @@ def critSub(matchobj):
             if c:
                 damage = f"{damage}+{c}"
 
+        if autoctx.maxdmg or maxdmg:
```

Comment:
why the check for automation context? It's never set anywhere and doesn't need to be stateful